pkg/network/node/pod.go

```diff
@@ -273,6 +271,7 @@ func (m *podManager) processRequest(request *cniserver.PodRequest) *cniserver.Po
 			}
 		}
 		if err != nil {
+			klog.Warningf("CNI_ADD %s/%s failed: %v", request.PodNamespace, request.PodName, err)
```

Comment:
@danwinship you can use `pk` here since that is a string that is already <namespace>/<name>